api/logbroker.proto

```diff
@@ -80,6 +85,10 @@ message LogMessage {
 
 	// Data is the raw log message, as generated by the application.
 	bytes data = 4;
+
+	// Details is a map of key value pairs representing additional log details
+	// that may have been returned from the task logger
+  repeated Details details = 5;
```

Comment:
indentation

I noticed that Docker calls this `attributes` - should we use that terminology? `details` also sounds okay but I'm wondering if there's a reason to use one over the other.